test/hummingbot/connector/exchange/kraken/test_kraken_api_order_book_data_source.py

```diff
@@ -0,0 +1,251 @@
+                    ],
+                    "leverage_sell": [
+                        2,
+                        3,
+                        4,
+                        5
+                    ],
+                    "fees": [
+                        [
+                            0,
+                            0.26
+                        ],
+                        [
+                            50000,
+                            0.24
+                        ],
+                    ],
+                    "fees_maker": [
+                        [
+                            0,
+                            0.16
+                        ],
+                        [
+                            50000,
+                            0.14
+                        ],
+                    ],
+                    "fee_volume_currency": "ZUSD",
+                    "margin_call": 80,
+                    "margin_stop": 40,
+                    "ordermin": "0.005"
+                },
+            }
+        }
+        return asset_pairs
+
+    def get_trade_data_mock(self) -> List:
+        trade_data = [
+            0,
+            [
+                [
+                    "5541.20000",
+                    "0.15850568",
+                    "1534614057.321597",
+                    "s",
+                    "l",
+                    ""
+                ],
+                [
+                    "6060.00000",
+                    "0.02455000",
+                    "1534614057.324998",
+                    "b",
+                    "l",
+                    ""
+                ]
+            ],
+            "trade",
+            f"{self.base_asset}/{self.quote_asset}"
+        ]
+        return trade_data
+
+    @aioresponses()
+    def test_get_last_traded_prices(self, mocked_api):
+        url = f"{CONSTANTS.BASE_URL}{CONSTANTS.TICKER_PATH_URL}"
+        regex_url = re.compile(f"^{url}".replace(".", r"\.").replace("?", r"\?"))
+        last_traded_price = Decimal("52641.10000")
+        resp = self.get_last_traded_prices_mock(last_trade_close=last_traded_price)
+        mocked_api.get(regex_url, body=json.dumps(resp))
+
+        ret = self.async_run_with_timeout(
+            KrakenAPIOrderBookDataSource.get_last_traded_prices(
+                trading_pairs=[self.trading_pair], throttler=self.throttler
+            )
+        )
+
+        self.assertIn(self.trading_pair, ret)
+        self.assertEqual(float(last_traded_price), ret[self.trading_pair])
+
+    @aioresponses()
+    def test_get_new_order_book(self, mocked_api):
+        url = f"{CONSTANTS.BASE_URL}{CONSTANTS.SNAPSHOT_PATH_URL}"
+        regex_url = re.compile(f"^{url}".replace(".", r"\.").replace("?", r"\?"))
+        resp = self.get_depth_mock()
+        mocked_api.get(regex_url, body=json.dumps(resp))
+
+        ret = self.async_run_with_timeout(self.data_source.get_new_order_book(self.trading_pair))
+
+        self.assertTrue(isinstance(ret, OrderBook))
```

Comment:
It would be better to assert something based on the status of the order book, instead of checking for the class